frontend/src/container/NewWidget/LeftContainer/WidgetGraph/index.tsx

```diff
@@ -41,6 +42,9 @@ function WidgetGraph({
 			selectedTime: selectedTime.enum,
 		},
 		selectedDashboard?.data?.version || DEFAULT_ENTITY_VERSION,
+		{
```

Comment:
We don't need a query-range API call for list panels?

Reply:
Why separate handling for List view only?

Reply:
So we have a navigation, to fetch the next and previous data from inside the panel, also data fetching is based on pagination, which other panels do not have. Handling it from the parent will be drilling down too many props for the child. That's the reason it was handled separately.